extensions/amp-access/0.1/amp-access.js

```diff
@@ -16,6 +16,7 @@
 
 import {actionServiceFor} from '../../../src/action';
 import {assertHttpsUrl} from '../../../src/url';
+import {installCidService} from '../../../src/service/cid-impl.js';
```

Comment:
… don't you think everyone will have `amp-analytics` anyway? Normally we assume that dependency.